sdk/guides/user-authentication.md

```diff
@@ -0,0 +1,216 @@
+---
+description: User Authentication
+---
+
+# User Authentication
+
+Connect and manage user wallet sessions in your dApp. This guide covers both **Wagmi** (recommended) and **vanilla JavaScript** approaches.
+
+<div style={{ 
+    display: 'flex', 
+    flexDirection: 'row', 
+    gap: '12px', 
+    marginBottom: '24px', 
+    overflow: 'hidden', 
+    overflowX: 'scroll', 
+    WebkitOverflowScrolling: 'touch',
+}}>
+    <img 
+        src={require("../_assets/quickstart-step-1.jpg").default} 
+        alt="Connect to MetaMask - Step 1" 
+        style={{ flex: '1', maxWidth: '35%', border: '1px solid #DCDCDC' }} 
+    />
+    <img 
+        src={require("../_assets/quickstart-step-2.jpg").default} 
+        alt="Connect to MetaMask - Step 2" 
+        style={{ flex: '1', maxWidth: '35%', border: '1px solid #DCDCDC' }} 
+    />
+    <img 
+        src={require("../_assets/quickstart-step-3.jpg").default} 
+        alt="Connect to MetaMask - Step 3" 
+        style={{ flex: '1', maxWidth: '35%', border: '1px solid #DCDCDC' }} 
+    />
+    <img 
+        src={require("../_assets/quickstart-step-4.jpg").default} 
+        alt="Connect to MetaMask - Step 4" 
+        style={{ flex: '1', maxWidth: '35%', border: '1px solid #DCDCDC' }} 
+    />
+    <img 
+        src={require("../_assets/quickstart-step-9.jpg").default} 
+        alt="Connect to MetaMask - Step 9" 
+        style={{ flex: '1', maxWidth: '35%', border: '1px solid #DCDCDC' }} 
+    />
+</div>
```

Comment:
I think it might be clearer to demonstrate this functionality in a video or gif, instead of several screenshots?

Reply:
Yeah, in retrospect I do agree. GIF works way better now.